content/en/docs/concepts/scheduling-eviction/taint-and-toleration.md

```diff
@@ -89,14 +89,10 @@ You can also use numeric comparison operators for threshold-based matching:
 * the `operator` is `Gt` (greater than) and the toleration value is greater than the taint value, or
 * the `operator` is `Lt` (less than) and the toleration value is less than the taint value.
 
-For numeric operators, both the toleration and taint values must be valid integers.
-If either value cannot be parsed as an integer, the toleration does not match.
+For numeric operators, both the toleration and taint values must be valid integers. If either value cannot be parsed as an integer, the toleration does not match.
```

Comment:
Why change this?
We encourage wrapping long "lines" for ease of reading and maintenance.